scripts/tools/check_includes_config.py

```diff
@@ -136,6 +136,7 @@
     'src/app/clusters/media-input-server/media-input-delegate.h': {'list'},
     'src/app/clusters/media-playback-server/media-playback-delegate.h': {'list'},
     'src/app/clusters/target-navigator-server/target-navigator-delegate.h': {'list'},
+    'src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.h': {'vector'},
```

Comment:
add a comment of why this is acceptable: probably webrtc assumes a large enough system that STL is acceptable.